line/line.mjs

```diff
@@ -1,27 +1,85 @@
-import * as utils from '../utilities/shared.mjs'
+import * as utils from '../utilities/shared.mjs';
 
-export async function findLineById(id = null) {
-  let line = null
+export async function findLineById(id = null, options = {}) {
+  let line = null;
+
+  if (id === null || id === undefined) {
+    return line;
+  }
 
   if (!utils.validateID(id)) {
-    return line
+    return line;
   }
 
   const mockPause = new Promise((resolve) => {
     setTimeout(() => {
-      resolve(null)
-    }, 1500)
-  })
+      resolve(null);
+    }, 1500);
+  });
 
   const linesArray = [
     { id: 123, text: "Hey TPEN Works on 123" },
-  ]
+  ];
 
-  line = linesArray.find((line) => line.id === id) || null
+  line = linesArray.find((line) => line.id === id) || null;
 
   if (line === null) {
-    line = await mockPause
+    line = await mockPause;
+  }
+
+  if (line !== null) {
+    if (options.text === 'blob') {
+      return { text: line.textualBody };
+    } else if (options.image === 'full') {
+      return { image: line.canvas };
+    } else if (options.image === 'line') {
+      return { image: `some line image URL for id ${id}` };
+    } else if (options.lookup) {
+      return { relatedDocument: `some ${options.lookup} document for id ${id}` };
+    } else {
+      const jsonResponse = {
+        '@context': line['@context'],
+        id: line.id,
+        '@type': line['@type'],
+        creator: line.creator,
+        textualBody: line.text,
+        project: line.project,
+        canvas: line.canvas,
+        layer: line.layer,
+        viewer: line.viewer,
+        license: line.license
+      };
+
+      if (options.view === 'xml') {
+        return generateXML(line);
```

Comment:
The return for this does not show an XML page, it shows JSON
![image](https://github.com/CenterForDigitalHumanities/TPEN-services/assets/3287006/663c4257-b64c-40e7-8d0d-f849ed535015)

Reply:
Uodated code shows json